Formula/minizip-ng.rb

```diff
@@ -19,9 +19,12 @@ class MinizipNg < Formula
   depends_on "zstd"
 
   uses_from_macos "bzip2"
-  uses_from_macos "libiconv"
   uses_from_macos "zlib"
 
+  on_macos do
+    depends_on "libiconv"
+  end
```

Comment:
Wait, why are we adding a dependency? If this is needed, then we also need a `revision` bump.

Reply:
Good question - actually I think this isn't needed. Brain fart - let me try removing it.